lib/binaries.sh

```diff
@@ -7,6 +7,33 @@ needs_resolution() {
   fi
 }
 
+install_yarn() {
+  local dir="$1"
+  local version=${2:-latest}
+
+  if [ "$version" = 'latest' ]; then
+    local download_url="https://yarnpkg.com/latest.tar.gz"
+  else
+    local download_url="https://yarnpkg.com/downloads/$version/yarn-v$version.tar.gz"
```

Comment:
I believe this causes an issue when yarn version is specified as `^0.17.8`. Currently our deploy is failing because it fails to resolve the url correctly.